GLideN64/src/Graphics/OpenGLContext/GLFunctions.cpp

```diff
@@ -342,7 +342,9 @@ extern "C" void initGLFunctions()
 	ASSIGN_PROC_ADR(PFNGLDELETEFRAMEBUFFERSPROC, DeleteFramebuffers);
 	ASSIGN_PROC_ADR(PFNGLGENFRAMEBUFFERSPROC, GenFramebuffers);
 	ASSIGN_PROC_ADR(PFNGLFRAMEBUFFERTEXTURE2DPROC, FramebufferTexture2D);
+#ifndef __APPLE__
 	ASSIGN_PROC_ADR(PFNGLTEXIMAGE2DMULTISAMPLEPROC, TexImage2DMultisample);
```

Comment:
Use ifdef apple GL_GET_PROC_ADR
If it doesnt resolve, let it null it via getproc
We don't wanna skip assignment here.

Reply:
Actually nvm, assign proc adr should work since it assigns to `rgl` and there it's jst a nop.
Why was this changed?

Reply:
```
GLideN64/src/Graphics/OpenGLContext/GLFunctions.cpp:345:2: error: assigning to
      'PFNGLTEXIMAGE2DMULTISAMPLEPROC'
      (aka 'void (*)(unsigned int, int, int, int, int, unsigned char)') from incompatible type 'void
      (GLenum, GLsizei, GLenum, GLsizei, GLsizei, GLboolean)' (aka 'void (unsigned int, int, unsigned
      int, int, int, unsigned char)'): type mismatch at 3rd parameter ('GLint' (aka 'int') vs 'GLenum'
      (aka 'unsigned int'))
        ASSIGN_PROC_ADR(PFNGLTEXIMAGE2DMULTISAMPLEPROC, TexImage2DMultisample);
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        ```
        
  ^ if not changed